harness/determined/pytorch/dsat/_run_dsat.py

```diff
@@ -18,7 +18,7 @@ def build_exp_conf_from_args(args: argparse.Namespace) -> _dsat_search_method.Ba
     exp_config = _utils.get_dict_from_yaml_or_json_path(
         args.config_path
     )  # add the search runner's experiment id to the description of the corresonding Trial
-    additional_description = f"Generated by experiment {args.experiment_id}"
+    additional_description = f"(#{args.experiment_id}) generated"
     existing_description = exp_config.get("description")
     if existing_description is not None:
         exp_config["description"] += "(" + additional_description + ")"
```

Comment:
In the `existing_description is not None` case, this will lead to double parenthesis due to the parens in `additional_description`, right?

Reply:
Thats true... I also wanted the number to be the to the far left so you don't have to mess with the table to see it. I can fix it up if you like

Reply:
happy for it to be on the left, should just avoid the double parens